MapboxNavigation/NavigationMapView.swift

```diff
@@ -1071,7 +1074,9 @@ open class NavigationMapView: MLNMapView, UIGestureRecognizerDelegate {
      */
     @objc public func setOverheadCameraView(from userLocation: CLLocationCoordinate2D, along coordinates: [CLLocationCoordinate2D], for bounds: UIEdgeInsets) {
         self.isAnimatingToOverheadMode = true
-        let slicedLine = Polyline(coordinates).sliced(from: userLocation).coordinates
+        guard let slicedLine = LineString(coordinates).sliced(from: userLocation)?.coordinates else {
```

Comment:
In which cases does LineString.sliced return nil in this case?

Reply:
Same cause as my previous comment - `sliced` will now return `nil` rather than an empty `LineString` in the case that `coordinates` is empty.

The implications of this one are trickier though — it's possible that this might result in some change in behavior. My wager is that it's OK, since passing empty coordinates seems ill formed in the first place. 

And in that case, our response is a no-op, rather than exploding or something scary.

I've just now added a debug assert to build confidence that we're not inadvertently hitting that code path. 

What do you think?